fleet/fleet.go

```diff
@@ -217,3 +256,6 @@ func (f fleet) ipFromUnitState(unitState *schema.UnitState) (net.IP, error) {
 
 	return nil, maskAny(ipNotFoundError)
 }
+
+// TODO
```

Comment:
Even though I think `PrintStatus(UnitStatus) string` belongs to the fleet package I will just move it to the CLI package.